syntax = "proto3";

package Order;

message OrderMessage {
    string status = 1;
    double total_amount = 2;
    string created_at = 3;
    string updated_at = 4;
    repeated ItemMessage items = 5;
}

message ItemMessage {
    int32 product_id = 1;
    int32 quantity = 2;
    double price = 3;
}